web/src/features/panels/zone/EstimationCard.tsx

```diff
@@ -75,7 +75,9 @@ function getEstimationTranslation(
       : t(`estimation-card.${estimationMethod?.toLowerCase()}.${field}`);
 
   const genericTranslation = t(`estimation-card.estimated_generic_method.${field}`);
-  return exactTranslation.length > 0 ? exactTranslation : genericTranslation;
+  return exactTranslation.length > 0 && !exactTranslation.startsWith('estimation-card.')
+    ? exactTranslation
+    : genericTranslation;
```

Comment:
Do we even need the length call anymore? I assume it will always be above 0 now as it returns the identifier (for some reason).

Reply:
True